vms/volunteer/tests/test_volunteerProfile.py

```diff
@@ -117,40 +153,78 @@ def test_edit_profile(self):
         found_org = re.search(new_details[9], page_source)
         self.assertNotEqual(found_org, None)
 
-        # database check to ensure that profile has been updated
-        self.assertEqual(len(Volunteer.objects.all()), 1)
-        self.assertNotEqual(
-            len(
-                Volunteer.objects.filter(
-                    first_name=new_details[0],
-                    last_name=new_details[1],
-                    email=new_details[2],
-                    address=new_details[3],
-                    city=new_details[4],
-                    state=new_details[5],
-                    country=new_details[6],
-                    phone_number=new_details[7])), 0)
-
-    def test_upload_resume(self):
-        pass
-        '''
-        #Tested locally
+    def test_valid_upload_resume(self):
```

Comment:
@naman1901 @abhi20161997 @pavankm I couldn't find a solution that will make this pass and other similar test (where we upload resume) pass. Please give your suggestions over how to proceed with this. :confused:

Reply:
For now, go ahead with commenting this and creating an issue.

Reply:
Done @naman1901 Please review again.